sgx-tools/dcap_linux.go

```diff
@@ -0,0 +1,358 @@
+package main // import "github.com/inclavare-containers/sgx-tools"
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"github.com/go-restruct/restruct"
+	"github.com/golang/protobuf/proto"
+	"github.com/inclavare-containers/rune/libenclave/intelsgx"
+	pb "github.com/inclavare-containers/sgx-tools/proto"
+	"github.com/sirupsen/logrus"
+	"net"
+)
+
+const (
+	aesmdSocket         = "/var/run/aesmd/aesm.socket"
+	attKeyIdLength      = 256
+	nonceLength         = 16
+	qeReportInfoLength  = 960
+	timeOut             = 15000000
```

Comment:
Add a comment to describe its unit. Second?

Reply:
> Add a comment to describe its unit. Second?

OK.